Resources/Prototypes/Catalog/Fills/Lockers/wardrobe_job.yml

```diff
@@ -29,47 +53,173 @@
+       - id: ClothingBackpackScience
+         amount: 1
+       - id: ClothingShoesColorWhite
+         amount: 1
+       - id: ClothingOuterCoatLab
+         amount: 1
+       - id: ClothingUniformJumpskirtScientist
+         amount: 1
+       - id: ClothingEyesGlasses
+         prob: 0.5
+       - id: CrowbarRed #The jojoke - hit video game Half-Life
+         prob: 0.1
+   
+   
+ - type: entity
+   id: WardrobeBotanistFilled
+   suffix: Filled
+   parent: WardrobeBotanist
+   description: "Plant yourself among the plant men with these 100% natural plant-derived clothes."
+   components:
+   - type: StorageFill
+     contents:
+       - id: ClothingUniformJumpsuitHydroponics
+         amount: 1
+       - id: ClothingBackpackHydroponics
+         amount: 1
+       - id: ClothingShoesColorBrown
+         amount: 1
+       - id: ClothingOuterApronBotanist
+         amount: 1
+       - id: ClothingUniformJumpskirtHydroponics
+         amount: 1
+       - id: ClothingUniformOveralls
+         amount: 1
+       - id: ClothingHeadHatTrucker #the hat itself is a subtle joke
+         prob: 0.3
+       - id: ClothingHandsGlovesLeather
+         prob: 0.5
 
-# - type: entity
-#   id: WardrobeBotanistFilled
-#   suffix: Filled
-#   parent: WardrobeBotanist
+ - type: entity
+   id: WardrobeMedicalDoctorFilled
+   suffix: Filled
+   parent: WardrobeMedicalDoctor
+   description: "We've all played doctor before, now practice medicine."
+   components:
+   - type: StorageFill
+     contents:
+       - id: ClothingUniformJumpsuitMedicalDoctor
+         amount: 1
+       - id: ClothingBackpackMedical
+         amount: 1
+       - id: ClothingShoesColorWhite
+         amount: 1
+       - id: ClothingOuterCoatLab
+         amount: 1
+       - id: ClothingUniformJumpskirtMedicalDoctor
+         amount: 1
+       - id: ClothingHandsGlovesLatex
+         prob: 0.4
+       - id: ClothingEyesGlasses
+         prob: 0.3
 
-# - type: entity
-#   id: WardrobeMedicalDoctorFilled
-#   suffix: Filled
-#   parent: WardrobeMedicalDoctor
-#   components:
-#   - type: StorageFill
-#     contents:
-#       - id: ClothingHandsGlovesLatex
-#         prob: 0.4
+ - type: entity
+   id: WardrobeChapelFilled
+   suffix: Filled
+   parent: WardrobeChapel
+   description: "Modest is hottest."
+   components:
+   - type: StorageFill
+     contents: # todo chaplain backpack
+       - id: ClothingUniformJumpsuitChaplain
+         amount: 1
+       - id: ClothingShoesColorBlack
+         amount: 1
+       - id: ClothingUniformJumpskirtChaplain
+         amount: 1
+       - id: ClothingHeadHatHoodChaplainHood
+         amount: 1
+       - id: ClothingHeadHatHoodNunHood
+         amount: 1
+       - id: ClothingOuterHoodieChaplain
+         amount: 1
+       - id: ClothingHeadHatFez
+         prob: 0.5
+       - id: ClothingHeadHatPlaguedoctor
+         prob: 0.15
+       - id: ClothingHeadHatWitch
+         prob: 0.05
+       - id: ClothingHeadHatWitch1
+         prob: 0.05
```

Comment:
these should probably be in the chaplain vendor or w/e right?

Reply:
Do we have that? If it should probably be mapped.